infra/discovery/devops/es-cli.js

```diff
@@ -68,6 +64,43 @@ const rl = readline.createInterface({
   output: process.stdout
 })
 
+/**
+ * Creates an object composed of keys generated from the results of running
+ * each element of `collection` thru `iteratee`.
+ *
+ * @param  {Array} collection The collection to iterate over.
+ * @param  {Function} iteratee The iteratee to transform keys.
+ * @return {Object} Returns the composed aggregate object.
+ */
+function groupBy(collection, iteratee) {
```

Comment:
@sparrowDom Could we replace those with lodash ?

Reply:
Yeah that would be even better. The original reason we didn't use lodash was that we didn't want to introduce a new package dependancy at that time.